search-service/src/test/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeServiceTests.kt

```diff
@@ -33,6 +39,9 @@ class TemporalEntityAttributeServiceTests : TimescaleBasedTests() {
     @SpykBean
     private lateinit var temporalEntityAttributeService: TemporalEntityAttributeService
 
+    @MockkBean
+    private lateinit var attributeInstanceService: AttributeInstanceService
```

Comment:
you could add a few more checks on the calls to `attributeInstanceService.create(...)`

Reply:
I was also wondering how it worked before without this service being mocked?

Reply:
Maybe because the old create function uses the `databaseClient.insert()` method.